src/components/Markdown/amber.ts

```diff
@@ -31,7 +31,7 @@ const amber = (hljs: HLJSApi) => {
     }
     const keywords = {
         scope: 'keyword',
-        match: /\b(and|as|break|cd|continue|echo|else|fail|failed|for|from|fun|if|import|in|is|len|let|lines|loop|main|mv|nameof|not|or|pub|ref|return|silent|status|then|trust|unsafe)\b/
+        match: /\b(ref|to|error|status|if|echo|loop|for|in|silent|return|fun|else|break|continue|and|or|not|let|const|sh|main|failed|fail|unsafe|then|pub|import|from|as|nameof|is)\b/
```

Comment:
Should probably put this back into alphabetical order, for simpler comparison next time round.